ext/pdo/pdo_stmt.c

```diff
@@ -708,14 +708,38 @@ static void do_fetch_opt_finish(pdo_stmt_t *stmt, int free_ctor_agrs) /* {{{ */
 }
 /* }}} */
 
+static bool pdo_do_key_pair_fetch(pdo_stmt_t *stmt, enum pdo_fetch_orientation ori, zend_long offset, HashTable *container)
+{
+	if (!do_fetch_common(stmt, ori, offset)) {
+		return false;
+	}
+	if (stmt->column_count != 2) {
+		/* TODO: Error? */
```

Comment:
Isn't the line below an error already?

Reply:
Yeah copied and pasted the existing code, will remove.
```suggestion
```